ci-operator/config/openshift-eng/ocp-qe-perfscale-ci/scripts/create_node_density_heavy_jobs.sh

```
@@ -0,0 +1,88 @@
+
+# Create the new file with version replacements
+sed -e "s/\"${PRIOR_VERSION}\"/\"${TARGET_VERSION}\"/g" -e "s/variant: ${PRIOR_VERSION}-nightly-node-density-heavy/variant: ${TARGET_VERSION}-nightly-node-density-heavy/g" "$SOURCE_FILE" > "$TARGET_FILE"
+
+echo "Created: $TARGET_FILE"
+echo ""
+
+# Validate YAML syntax
+echo "Validating YAML syntax..."
+if command -v python3 &> /dev/null; then
+    python3 -c "import yaml; yaml.safe_load(open('$TARGET_FILE'))" 2>&1
+    if [ $? -ne 0 ]; then
```

Shellcheck (0.11.0)

[style] Check exit code directly with e.g. 'if ! mycmd;', not indirectly with $?.

(SC2181)